environments/centos-9/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM rockylinux:9
+
+# Used to link container image to the repo:
+# https://docs.github.com/en/free-pro-team@latest/packages/managing-container-images-with-github-container-registry/connecting-a-repository-to-a-container-image#connecting-a-repository-to-a-container-image-on-the-command-line
+LABEL org.opencontainers.image.source https://github.com/fullstaq-ruby/server-edition
+
+# If you make a change and you want to force users to re-pull the image
+# (e.g. when your change adds a feature that our scripts rely on, or is
+# breaking), then bump the version number in the `image_tag` file.
+
+RUN set -x && \
+    dnf install -y dnf-plugins-core epel-release && \
+    dnf install -y --enablerepo epel --enablerepo powertools \
```

Comment:
This doesn't work:

~~~
+ dnf install -y --enablerepo epel --enablerepo powertools findutils gcc gcc-c++ make patch bzip2 curl autoconf automake openssl-devel libyaml-devel libffi-devel readline-devel zlib-devel gdbm-devel ncurses-devel
Error: Unknown repo: 'powertools'
~~~

But also:

~~~
No match for argument: libyaml-devel
No match for argument: gdbm-devel
~~~

Reply:
Fixed by replacing `--enablerepo powertools` with `--enablerepo devel`.